src/OrchardCore.Modules/OrchardCore.Facebook/Login/Configuration/FacebookLoginConfiguration.cs

```diff
@@ -62,7 +68,7 @@ public void Configure(AuthenticationOptions options)
         public void Configure(string name, FacebookOptions options)
         {
             // Ignore OpenID Connect client handler instances that don't correspond to the instance managed by the OpenID module.
-            if (!string.Equals(name, FacebookDefaults.AuthenticationScheme))
+            if (!String.Equals(name, FacebookDefaults.AuthenticationScheme))
```

Comment:
Is this the standard? I've seen both types being used. Also interesting for https://github.com/OrchardCMS/OrchardCore/issues/7950.

Reply:
I have no preference, just to hide any VS warning

According to what we currently have in the `.editorconfig`

    # Use language keywords instead of framework type names for type references
    dotnet_style_predefined_type_for_locals_parameters_members = true : suggestion
    dotnet_style_predefined_type_for_member_access             = false : suggestion <=

Reply:
Yeah that's right. I don't like it though :D.